test/support/test_helpers.ex

```diff
@@ -40,6 +46,19 @@ defmodule Parser.TestHelpers do
     end
   end
 
+  def resolve_longest_hyphenated_commands(_bundle, cmd) do
+    cond do
+      cmd == "stacks-ls-prod" ->
+        {:command, {"cfn", cmd}}
+      cmd == "stacks-ls" ->
+        {:command, {"cfn", cmd}}
+      cmd == "ls-prod" ->
+        {:error, {:ambiguous, ["ec2", "cfn"]}}
+      true ->
+        {:error, :not_found}
+    end
```

Comment:
Would probably be nice and more efficient to use `case` here. Not a big deal though.